### PR TITLE
Fix NPM Dependencies

### PR DESCRIPTION
**Pull request by willmt80**
removed vast majority of low priority vulnerabilities and all but 1 high priority

Capture

### CODE REVIEW

**Comment by willmt80** [comment]
Apparently it is not as easy as updating as much as possible and forcing resolutions. Breaks Travis

**Comment by willmt80** [comment]
Travis error is **npm ERR! Invalid dependency type requested: alias**. The alias type is allowed in npm versions 6.9.0 and above. Our Travis build uses npm version 6.2.0.

File: .travis.yml [hunk 1]
```diff
@@ -1,5 +1,7 @@
 language: node_js
 node_js: 
   - 10.8
+before_install:
+  - npm i -g npm@6.9.0
```
Comment (jackblanc) [inline .travis.yml:5]: Hmm, I don't know what we want to have to do this ourselves. Why is this required?

File: package.json [hunk 2]
```diff
@@ -54,5 +54,8 @@
       "vue-cli-service lint",
       "git add"
     ]
+  },
+  "resolutions": {
```
Comment (jackblanc) [inline package.json:58]: What's this for?

File: .travis.yml (hunk 1 above)
Comment (willmt80) [reply .travis.yml:5]: A couple of the updated dependencies have aliases which were introduced in npm version 6.9.0 (which is a couple of years old). The version of Travis we are running uses outdated versions of node and npm (current recommended node is 12.18.3 with npm 6.14.6). We may be able to just change the version of node and remove that before_install clause. Ultimately what we should do depends on if we are using an older version of npm and node for a specific reason.

File: package.json (hunk 2 above)
Comment (willmt80) [reply package.json:58]: To force a version of the serialize-javascript library that had a high vulnerability. This was fixed the day after I committed these changes lol https://github.com/webpack-contrib/copy-webpack-plugin/releases/tag/v5.1.2

**Review by justinkonecny — APPROVED** [review]

File: .travis.yml (hunk 1 above)
Comment (jackblanc) [reply .travis.yml:5]: Circling back here, can we just change the node version above?
